tasks/DCPowerCubeTask.cpp

```diff
@@ -22,6 +22,7 @@ bool DCPowerCubeTask::configureHook()
     }
 
     m_driver = DCPowerCube(_device_id.get());
+    m_io_read_timeout = _io_read_timeout.get();
```

Comment:
Set a decent default value in the constructor ! https://www.rock-robotics.org/rock-and-syskit/components/interface.html#properties